engine/index.js

```diff
@@ -10,6 +10,7 @@ import BrowserParser from './browserParser.js';
 import FirefoxVersionParser from './firefoxVersionParser.js';
 import CanIUseParser from './canIUseParser.js';
 import cache from './cache.js';
+import redis from './redis-helper.js';
```

Comment:
It might grow - I have yet no idea how to make it better with promises, but I bet I will. I will add some explanation inside the helper file (along with license)